geomesa-gs-plugin/geomesa-hbase-gs-plugin/pom.xml

```diff
@@ -13,17 +13,13 @@
     <parent>
         <artifactId>geomesa-gs-plugin</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
-        <version>1.2.0-SNAPSHOT</version>
+        <version>1.2.2-SNAPSHOT</version>
```

Comment:
I think parts of them were moved out for license reasons, but the core bundles are still here.